src/big/big_float.cr

```diff
@@ -278,30 +286,62 @@ struct BigFloat < Float
     mpf
   end
 
+  # :nodoc:
+  def precision
+    LibGMP.mpf_get_prec(self)
+  end
```

Comment:
Similarly, this also seems to be only used in specs, maybe this could be just implemented in the spec helper directly? It's just a single call either way.